### PR TITLE
adjust changelog for release v0.7.0-rc.0

### PR DESCRIPTION
**Pull request by wrongerror**
Signed-off-by: wrongerror <wangyifei@kubesphere.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,24 +20,96 @@
...
+#### Features
+- Support OpenFunction framework. [functions-framework-java#1](https://github.com/OpenFunction/functions-framework-java/pull/1)
+
+### openfunction.dev
+Renaming OpenFunction' website repository to openfunction.dev.
+
+Docs have been refactored and updated with all the new features and changes of this release, see [OpenFunction docs](https://openfunction.dev/docs/).
+
+### charts
+Now you can install OpenFunction and all its dependencies with helm charts.
```
Comment (benjaminhuo) [inline CHANGELOG.md:98]: we can add something like upgrade dapr from v1.5.x to 1.8.3

**Review by tpiperatgod — APPROVED** [review]

**Review by lizzzcai — APPROVED** [review]
looks good for ff-go

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,24 +20,103 @@
...
+#### **TL;DR**
+```shell
+helm repo add openfunction https://openfunction.github.io/charts/
+helm repo update
+helm install openfunction openfunction/openfunction -n openfunction --create-namespace
+```
+
+#### Component Upgrade
+- knative-serving from v1.0.1 to v1.3.2
```
Comment (benjaminhuo) [inline CHANGELOG.md:107]:
```suggestion
- Upgrade knative-serving from v1.0.1 to v1.3.2
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,24 +20,103 @@
...
+- shipwright-build from v0.6.1 to 0.10.0
```
Comment (benjaminhuo) [inline CHANGELOG.md:108]:
```suggestion
- Upgrade shipwright-build from v0.6.1 to 0.10.0
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,24 +20,103 @@
...
+- tekton-pipelines from v0.30.0 to v 0.37.2
```
Comment (benjaminhuo) [inline CHANGELOG.md:109]:
```suggestion
- Upgrade tekton-pipelines from v0.30.0 to v 0.37.2
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -20,24 +20,103 @@
...
+- keda from v2.4.0 to v2.7.1
```
Comment (benjaminhuo) [inline CHANGELOG.md:110]:
```suggestion
- Upgrade keda from v2.4.0 to v2.7.1
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -20,24 +20,103 @@
...
+- dapr from v1.5.1 to v1.8.3
```
Comment (benjaminhuo) [inline CHANGELOG.md:111]:
```suggestion
- Upgrade dapr from v1.5.1 to v1.8.3
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Pull request merged**